app/controllers/students_controller.rb

```diff
@@ -1,6 +1,8 @@
+require 'google/calendar/calendar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.